tools/ARIAtools/unwrapStitching.py

```diff
@@ -62,6 +64,7 @@ def __init__(self):
         self.outFileConnComp = './connCompMerged'
         self.outputFormat='ENVI'
         self.verbose=False
+        log.setLevel('DEBUG') if self.verbose else ''
```

Comment:
Codacy found an issue: [Expression "log.setLevel('DEBUG') if self.verbose else ''" is assigned to nothing](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=6018914)